### PR TITLE
Post ga fixes

### PR DESCRIPTION
**Pull request by apinnick**
Doc previews:  

http://file.tlv.redhat.com/~apinnick/post-ga_fixes/migration/migrating-openshift-3-to-4.html#migration-configuring-cors-3_migrating-openshift-3-to-4

http://file.tlv.redhat.com/~apinnick/post-ga_fixes/migration/migrating-openshift-3-to-4.html#installing-migration-operator-manually_migrating-openshift-3-to-4

To fix the following bugs (verified by QE):

https://bugzilla.redhat.com/show_bug.cgi?id=1767951
https://bugzilla.redhat.com/show_bug.cgi?id=1767237
https://bugzilla.redhat.com/show_bug.cgi?id=1767948
https://bugzilla.redhat.com/show_bug.cgi?id=1767943

### CODE REVIEW

**Comment by apinnick** [comment]
@bergerhoffer This is ready for review. All bugs have been verified by QE. Thanks!

File: modules/migration-configuring-cors-3.adoc [hunk 1]
```diff
@@ -48,10 +48,10 @@ $ curl -v -k -X OPTIONS \
 "<cluster_url>/apis/migration.openshift.io/v1alpha1/namespaces/openshift-migration/migclusters" \ <1>
 -H "Access-Control-Request-Method: GET" \
 -H "Access-Control-Request-Headers: authorization" \
--H "Origin: https://<CAM_tool_url>" <2>
+-H "Origin: https://<CAM_web_console_url>" <2>
 ----
-<1> Update the source cluster URL.
-<2> Update the CAM tool URL.
+<1> Specify your CORS configured cluster's URL.
```
Comment (bergerhoffer) [inline modules/migration-configuring-cors-3.adoc]: Suggest rewording, this was a little hard to understand at first read through. Maybe "Specify the URL for your CORS-configured cluster."

File: modules/migration-configuring-cors-4.adoc [hunk 2]
```diff
@@ -60,10 +60,10 @@ $ curl -v -k -X OPTIONS \
 "<cluster_url>/apis/migration.openshift.io/v1alpha1/namespaces/openshift-migration/migclusters" \ <1>
 -H "Access-Control-Request-Method: GET" \
 -H "Access-Control-Request-Headers: authorization" \
--H "Origin: https://<CAM_tool_url>" <2>
+-H "Origin: https://<CAM_web_console_url>" <2>
 ----
-<1> `<cluster_url>`: Update the cluster URL.
-<2> `<CAM_tool_url>`: Update the CAM tool URL.
+<1> Specify your CORS configured cluster's URL.
```
Comment (bergerhoffer) [inline modules/migration-configuring-cors-4.adoc]: Same comment as the other similar wording.

**Review by bergerhoffer — COMMENTED** [review]
Just a wording suggestion, otherwise LGTM!

**Comment by bergerhoffer** [comment]
LGTM!
